src/documentation/language/datatypes.malloynb

```diff
@@ -126,6 +126,15 @@ Ranges may be used in conjunction with the [apply operator](apply.malloynb) to t
 
 In the future, other ranges may be allowed, such as `string` ranges.
 
+### Duration
+
+Durations can be added to timestamps and dates.
+
+* `@2001-02-03 04:05 + 5 minutes`
+* `@2001-02-03 04:05 + myHours hours`
+* `@2021-01-01 + 5 weeks`
+* `@0000-01-01 + myYear years + (daysIntoMyYear - 1) days`
+
```

Comment:
Thanks for starting this.

A few notes.

1) Durations can be added or subtracted.
2) The legal units for durations are, i think, microseconds, milliseconds, seconds, minutes, hours, days, weeks.
3) SQL durations, if the database has them, are not imported and useful as Malloy durations.

Reply:
I also modified point3 for clarity.